package.json

```diff
@@ -15,6 +15,7 @@
     "build": "okta-sdk-generator -t templates/ -o . && ./utils/maintain-banners.js",
     "predocs": "rm -rf ./jsdocs && mkdir jsdocs/ && ./utils/make-jsdoc-readme.js > ./jsdocs/jsdoc-temp.md",
     "docs": "./node_modules/.bin/jsdoc src/ -c ./docs/config.json -d ./jsdocs/ -P ./package.json -R ./jsdocs/jsdoc-temp.md -r",
+    "test:unit": "mocha test/unit/**/*.js --no-timeouts",
```

Comment:
As an aside, we've been switching to Jest elsewhere - it's pretty nice (can go into more details why if you're curious).

Reply:
Yeah jest is cool!  We're using it on some of the front-end libs.  We could consider switching to it for this library at some point.